src/components-examples/angular/textarea/textarea-forms/textarea-forms-example.html

```diff
@@ -11,16 +11,13 @@ <h4>Model</h4>
 <pre>{{ textarea | json }}</pre>
 
 <h4>Properties</h4>
-<p>
-  <sbb-form-field label="Min Length">
-    <input sbbInput type="number" [(ngModel)]="minlength" />
-  </sbb-form-field>
-</p>
-<p>
-  <sbb-form-field label="Max Length">
-    <input sbbInput type="number" [(ngModel)]="maxlength" />
-  </sbb-form-field>
-</p>
+<sbb-form-field label="Min Length">
+  <input [(ngModel)]="minlength" sbbInput type="number" />
+</sbb-form-field>
+<br />
```

Comment:
We should not use br for separation
Maybe we need to define a class similar to .sbb-checkbox-group-vertical?